### ISSUE TITLE
Hierarchy definition

### ISSUE THREAD

**Issue by karthikdatta98** [RomanPlaud/revisitingHTC#1]
Can I define a hierarchy and not provide any training examples for it?
The problem is that I have a huge hierarchy but not all classes have data. So when I define the taxonomy, can I mention the hierarchy for all the levels and not have training examples for few classes?

**Comment by RomanPlaud** [comment]
Yes i really don't think it is a problem. You can definitely do what your are saying
________________________________
De : karthikdatta98 ***@***.***>
Envoyé : jeudi 5 décembre 2024 20:49
À : RomanPlaud/revisitingHTC ***@***.***>
Cc : Subscribed ***@***.***>
Objet : [RomanPlaud/revisitingHTC] Hierarchy definition (Issue #1)


Can I define a hierarchy and not provide any training examples for it?
The problem is that I have a huge hierarchy but not all classes have data. So when I define the taxonomy, can I mention the hierarchy for all the levels and not have training examples for few classes?
